scripts/snyk.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# When packages change their dependency tree, we want to run snyk monitor. This
+# sends the updated dependency tree to snyk for monitoring.  We want to do this
+# when a commit has been made to master, not on PRs because the dependency tree
```

Comment:
I'm unsure if running **only** on `master` is the best decision here. As we add new packages, we want to make sure we find the dependency vulnerability before it gets merged.

As an alternative, we could have this script listen to `master` and a custom branch `snyk-*`. When we wish to add a new package, our flow will be to validate the dependencies via the `synk-some-description-here` branch, and (if passing), merge to `master`.

Reply:
> I'm unsure if running only on master is the best decision here. As we add new packages, we want to make sure we find the dependency vulnerability before it gets merged.

That concern will be handled separately from this PR.  There is a separate `snyk test` command that can be used for that purpose.  This PR is focused on updating our dep tree with snyk when it does change.